compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -163,13 +163,29 @@ private sealed trait WarningSettings:
     name = "-Wunused",
     helpArg = "warning",
     descr = "Enable or disable specific `unused` warnings",
-    choices = List("nowarn", "all"),
+    choices = List("nowarn", "all", "imports", "locals", "privates", "patvars", "explicits", "implicits", "params", "linted"),
```

Comment:
For Scala 3 there should probably be an option `givens` as an alias for `implicits`?

Also, does `linted` make sense when the compiler doesn't have `-Xlint`? Or should `-Xlint:unused` be added along this PR?

Reply:
A `-Wunused:givens` would make sens to me in the context of Scala 3.

For the `linted`, I just added it for the compatibility with Scala 2 options. I don't know if this options is used on its own.